cmd/powcfg/ratelim/ratelim.go

```diff
@@ -0,0 +1,47 @@
+package ratelim
```

Comment:
I decided to do a generic rate limiter. 
This is an idea that we applied in many places in Powergate, but never re-used as an internal library.
Eventually, we move this lib to main Powergate and use it.

Reply:
Sweet